guidelines.md

```diff
@@ -101,3 +101,61 @@ As an example, consider minimum and maximum length of the reservation:
 * MinLength `P0Y0M1DT0H0M0S`
 * MaxLength `P0Y0M3DT0H0M0S`
 
+# Mews Certification
+
+Before going live in the production environment all connections must be certified by Mews.
+
+### Pre-Certification
+
+- Sign in to the Mews Connector API demo account. Credentials for this can be found in the [guidelines](https://mews-systems.gitbook.io/connector-api/guidelines#authentication) section.
+- Create a unique access token. The client token will always be `E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D`.
+  - Go to **Main menu** > **Settings** > **Marketplace**.
+  - Search for **connector API Demo client**.
+  - You can search the Marketplace by using either the categories section (the application you need is in the Connectivity section), or by pressing Ctrl (Windows) or Command (Mac) + F to search by name.
+  - Select **see more** > **ok**.
+    - If you're having trouble, check the [Mews guide](https://help.mewssystems.com/reference#about-marketplace) for more information on setting up the connection.
```

Comment:
All these lines can be written in the client demo description, or in pictures, video, ...